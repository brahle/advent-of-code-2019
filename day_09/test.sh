#!/bin/bash
./Day09/Day09/bin/Debug/netcoreapp3.1/Day09.exe < test/test.$1.in | grep -e "\(OUTPUT\|DONE\)" > out

diff -q -b test/test.$1.out out
if [ $? -ne 0 ]
then
    echo "EXPECTED: "
    cat test/test.$1.out

    echo ""
    echo "GOT: "
    cat out
    exit 1
fi
