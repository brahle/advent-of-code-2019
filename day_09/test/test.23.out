OUTPUT: 109
OUTPUT: 1
OUTPUT: 204
OUTPUT: -1
OUTPUT: 1001
OUTPUT: 100
OUTPUT: 1
OUTPUT: 100
OUTPUT: 1008
OUTPUT: 100
OUTPUT: 16
OUTPUT: 101
OUTPUT: 1006
OUTPUT: 101
OUTPUT: 0
OUTPUT: 99
DONE 109
